ansible/roles/kong-api/defaults/main.yml

```diff
@@ -5946,6 +5946,24 @@ kong_apis:
     - name: request-size-limiting
       config.allowed_payload_size: "{{ small_request_size_limit }}"
 
+  - name: userList
```

Comment:
Is this API documented @anmol2302

Reply:
@SMYALTAMASH PR sent to dock repo: https://github.com/project-sunbird/sunbird.org-docs/pull/736